### PR TITLE
go-mysqlx, go-binlog: regenerate code using proto 0.5

### PR DESCRIPTION
**Pull request by zhexuany**
#90 forgot to regenerate `go-mysqlx` and `go-binlog`.

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: generate-go.sh [hunk 1]
```diff
@@ -2,9 +2,19 @@ cd proto
...
-protoc -I.:${GOGO_ROOT}:${GOGO_ROOT}/protobuf --gofast_out=../go-tipb *.proto
-cd ../go-tipb
-sed -i.bak -E 's/import _ \"gogoproto\"//g' *.pb.go
-sed -i.bak -E 's/import fmt \"fmt\"//g' *.pb.go
-rm -f *.bak
-goimports -w *.pb.go
+if [ ! -d $GOGO_ROOT ]; then
+	echo "please use the following command to get specific version of gogo.\n\n"
+	echo "go get -u github.com/gogo/protobuf/protoc-gen-gofast"
+	echo "cd /Users/zhexuany/repo/go/src/github.com/gogo/protobuf"
```
Comment (shenli) [inline generate-go.sh]: Why use your own path?

Comment (zhexuany) [reply generate-go.sh]: already addressed.

File: generate-go.sh [hunk 2]
```diff
@@ -2,9 +2,19 @@ cd proto
...
-sed -i.bak -E 's/import fmt \"fmt\"//g' *.pb.go
-rm -f *.bak
-goimports -w *.pb.go
+if [ ! -d $GOGO_ROOT ]; then
+	echo "please use the following command to get specific version of gogo.\n\n"
+	echo "go get -u github.com/gogo/protobuf/protoc-gen-gofast"
+	echo "cd ${GOPATH}/src/github.com/gogo/protobuf"
+	echo "git checkout v0.5"
+	echo "rm ${GOPATH}/bin/protoc-gen-gofast"
+	echo "go get github.com/gogo/protobuf/protoc-gen-gofast"
```
Comment (shenli) [inline generate-go.sh]: Should we return some error? Like `exit 1`

Comment (zhexuany) [reply generate-go.sh]: no need. This logic is in `if` branch. If such directory is not existed, `else` branch will not be triggered.

**Pull request merged**